maintainers/maintainer-list.nix

```diff
@@ -8352,6 +8352,13 @@
     githubId = 265084;
     name = "Jason O'Conal";
   };
+  loveisgrief = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Done in 7df27f1